src/vos/tests/vts_mvcc.c

```diff
@@ -1486,6 +1492,25 @@ uncertainty_check_exec_one(struct io_test_args *arg, int i, int j, bool empty,
 		print_message("FAILED: "DF_CASE"\n",
 			      DP_CASE(i, j, empty, w, wp, we, commit, a, ap, ae,
 				      bound, mvcc_arg->i));
+
+	if (!daos_is_zero_dti(&wtx->th_saved_xid)) {
+		if (wtx->th_skip_commit)
```

Comment:
if skip_commit, then commit ?

Reply:
skip_commit means that we does not want to commit related DTX to generate some special test condition for the test. But after the test, it is OK to commit it since it is done successfully.